src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -13,36 +26,107 @@
         }
 
         public static class CANMotor {
+            // Intake
+            public static final int INTAKE_ROLLER_MOTOR = 10;
+            public static final int INTAKE_FUNNEL_MOTOR = 5;
+            // Indexer
+            public static final int LIFT_BELT_MOTOR = 6;
+            // Flywheel
+            public static final int FLYWHEEL_MOTOR_A = 2;
+            public static final int FLYWHEEL_MOTOR_B = 1;
+            // Shooter
+            public static final int RUN_UP_BELT_MOTOR = 8;
         }
 
-        public static class PWM {
+        public static class PWM { // TODO: CHANGE CONSTS
         }
 
-        public static class CAN {
+        public static class CAN { // TODO: CHANGE CONSTS
+            public static final int HOOD_MOTOR = -1;
+            public static final int HOOD_ENCODER = -1;
```

Comment:
No hood encoder—we'll use the encoder that's built into the falcon that's driving the hood.